.appveyor.yml

```diff
@@ -0,0 +1,18 @@
+os: Visual Studio 2017
+
+install:
+  - appveyor DownloadFile https://win.rustup.rs/ -FileName rustup-init.exe
+  - rustup-init -y --default-toolchain stable --default-host x86_64-pc-windows-msvc
+  - set PATH=%PATH%;%USERPROFILE%\.cargo\bin
+  - set RUST_BACKTRACE=1
+  - rustc -vV
+  - cargo -vV
+  # For build Compiling jni v0.10.2 
```

Comment:
First of all, I would change this:
`Copy jvm.lib in specific directory to compile on Windows` (notice lack of specific jni version mentioning).
Secondly, shouldn't we change our `build.rs` instead? Probably with help of https://github.com/jni-rs/jni-rs/pull/97

Reply:
In build.rs set correct name jvm.dll ( C:\Progra~1\Java\jdk1.8.0\jre\bin\server\jvm.dll present )
But it want jvm.lib
`note: LINK : fatal error LNK1181: cannot open input file 'jvm.lib'`

Reply:
An error occurs during the invocation test. Build is successful without copying the library

Reply:
If I understand correctly, it fails even if we copy that library (the error message is not clear):

```
test result: ok. 2 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out
     Running target\debug\deps\java_integers-ac0191d3d6d9a53d.exe
error: test failed, to rerun pass '--test java_integers'
```

I'd just not run these tests till we enable proper support of invocation API on Windows.

Reply:
Is `jvm.lib` a so-called 'import' library: https://stackoverflow.com/a/22708457/1792188 and https://stackoverflow.com/a/2337985/1792188?

Reply:
Without  test --invocation, build and test --backtrace success .
When run test --invocation without copy jvm.lib to ..jre\bin\server\ 
```
Compiling jni v0.10.2 (file:///C:/projects/jni-rs)
error: linking with `C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.15.26726\bin\HostX64\x64\link.exe` failed: exit code: 1181
.....
= note: LINK : fatal error LNK1181: cannot open input file 'jvm.lib'
```
When copyed  jvm.lib to ..jre\bin\server\ and run test  --invocation, got error 
```
Running target\debug\deps\java_integers-ac0191d3d6d9a53d.exe
error: test failed, to rerun pass '--test java_integers'
Command exited with code -1073741515
```

Reply:
Default:
jvm.lib located C:\Progra~1\Java\jdk1.8.0\lib\jvm.lib
jvm.dll located C:\Progra~1\Java\jdk1.8.0\jre\bin\server\jvm.dll